Classes/Issues/IssuesViewController.swift

```diff
@@ -642,5 +648,59 @@ final class IssuesViewController: MessageViewController,
         guard let issueType = self.issueType else { return }
         presentLabels(client: client, owner: owner, repo: repo, label: label, type: issueType)
     }
+    
+    // MARK: EditIssueTitleViewControllerDelegate
 
+    func sendEditTitleRequest(newTitle: String, viewController: EditIssueTitleViewController) {
```

Comment:
Why not just put all the sending stuff _inside_ the edit VC? Can include the cache updating stuff since the issue VC listens for model changes and will reload. Would totally eliminate the need for this delegate API and keep things really separated.